### ISSUE TITLE
Change doctor's queryset filter

### ISSUE THREAD

**Issue by germain-gg** [who-pays-your-doctor/whopaysthisdoctor.org#12]
Durning NHSHD2019 we changed the query to list all of the doctors. Due to time constraint we changed to
```
models.Doctor.objects.all()
```

It used to be 

```
models.Doctor.objects.filter(
  declaration__dt_created__lte=dt.datetime.now()-dt.timedelta(hours=1)
)
```

The new logic should only list doctors that have at least one declaration or at least one imported directorship.

**Comment by samsmith** [comment]
on the live site, there is a deliberate lag on updates being visible, so people who make a mistake don't panic and can email us to fix it before it appears.

**Comment by germain-gg** [comment]
Yup definitely, I wanted to keep the filter, I just didn't have enough time to get that working before the demo. Definitely needs to be improved before we make any update on the live site
